ibm/service/power/resource_ibm_pi_network_port_attach.go

```diff
@@ -64,6 +71,7 @@ func ResourceIBMPINetworkPortAttach() *schema.Resource {
 				Computed: true,
 			},
 			Arg_UserTags: {
+				Computed:    true,
```

Comment:
In this file `Arg_UserTags` is a ForceNew since a CRN is not returned, it does not need the customize diff or computed attribute

Reply:
On second thought, we can probably leave this in.